portal/build.sbt

```diff
@@ -0,0 +1,5 @@
+enablePlugins(JavaServerAppPackaging)
```

Comment:
Why do you think it is better to use `JavaServerAppPackaging` instead of `JavaAppPackaging`?

Reply:
I think that you are right, we don't need the features of server archetype yet. Thank you!